### PR TITLE
chore(common): Check in crowdin strings for km for final Beta sprint

### PR DESCRIPTION
**Pull request by darcywong00**
Check in latest km strings for the final Beta sprint.
Note: 2 iOS files were unchanged. The other two still show as "binary" files...

### CODE REVIEW

**Comment by jahorton** [comment]
> Note: 2 iOS files were unchanged. The other two still show as "binary" files...

I wonder if they're automatically downloading from Crowdin in UTF-16LE or something like that.  Would be a pain if we constantly have to reconvert the files into UTF-8, but that may be the case...

File: ios/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# localizable strings for crowdin
+*.strings diff=localizablestrings
+[diff "localizablestrings"]
+    textconv = "iconv -f utf-16 -t utf-8"
```
Comment (darcywong00) [inline ios/.gitattributes:4]: I'll likely revert this file. Was trying to configure so GH would diff the files, but no joy

**Comment by mcdurdin** [comment]
Can you reconvert those two UTF-16 .strings files into UTF-8 (e.g. using VS Code?)

**Comment by jahorton** [comment]
Seeing an issue on my dev machine here:

Screen Shot 2021-03-15 at 10 20 27 AM

Something's off with the git-attributes.

**Comment by darcywong00** [comment]
Git chokes on the rebase, so abandoning this one

**Pull request closed without merging**